### PR TITLE
Added instructions to enable MySQL interface in Cloud

### PR DESCRIPTION
**Pull request by Ryado**
Added instructions to enable MySQL interface in Cloud

### Changelog category (leave one):

- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
This is an automated comment for commit 1a42d71514570214d683de1aef93b0090363dfde with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/53554/1a42d71514570214d683de1aef93b0090363dfde/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Review by mshustov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
@Ryado @mshustov tests have failed.

**Comment by mshustov** [comment]
@Felixoid how can we run the failing `DocsCheck / StyleCheck (pull_request)` locally? https://github.com/ClickHouse/ClickHouse/actions/runs/5904106233/job/16047913088?pr=53554

**Comment by rschu1ze** [comment]
The style check scripts that run in CI are in `utils/check-style/`, e.g. the spell check: `utils/check-style/check-doc-aspell`.

But in this case, you can just follow the [instructions](https://s3.amazonaws.com/clickhouse-test-reports/53554/1a42d71514570214d683de1aef93b0090363dfde/style_check/docs_spelling_output.txt) given in CI:

```
====== /ClickHouse/docs/en/interfaces/mysql.md ======
exisitng
====== Errors found ======
To exclude some words add them to the dictionary file "/ClickHouse/utils/check-style/aspell-ignore/en/aspell-dict.txt"
You can also run ./check-doc-aspell -i to see the errors interactively and fix them or add to the dictionary file
```

**Comment by Felixoid** [comment]
The best way to run it locally is docker, the command is in job's logs https://github.com/ClickHouse/ClickHouse/actions/runs/5904106233/job/16015524579#step:6:24

File: docs/en/interfaces/mysql.md [hunk 1]
```diff
@@ -6,7 +6,34 @@ sidebar_label: MySQL Interface
 
 # MySQL Interface
 
-ClickHouse supports MySQL wire protocol. To enable the MySQL wire protocol, add the [mysql_port](../operations/server-configuration-parameters/settings.md#server_configuration_parameters-mysql_port) setting to your server's configuration file. For example, you could define the port in a new XML file in your `config.d` folder:
+ClickHouse supports the MySQL wire protocol. This allow tools that are MySQL compatible to interact with ClickHouse seamlessly (eg. [Looker Studio](../integrations/data-visualization/looker-studio-and-clickhouse.md)).
```
[the remark this answers is not in the document]
Comment (Ryado) [reply docs/en/interfaces/mysql.md]: ✅

[the remark this answers is not in the document]
Comment (Ryado) [reply docs/en/interfaces/mysql.md]: ✅

File: docs/en/interfaces/mysql.md [hunk 2]
```diff
@@ -6,7 +6,34 @@ sidebar_label: MySQL Interface
...
+2. Toggle the switch to enable the MySQL interface for this specific service. This will prompt you to enter the default password.
+
+![Connection screen - Prompt MySQL](./images/mysql4.png)
+
+3. After entering the password, you will get prompted the MySQL connection string for this service
+![Connection screen -  MySQL Enabled](./images/mysql5.png)
+
+## Enabling the MySQL Interface On Self-managed ClickHouse
+
+Add the [mysql_port](../operations/server-configuration-parameters/settings.md#server_configuration_parameters-mysql_port) setting to your server's configuration file. For example, you could define the port in a new XML file in your `config.d` folder:
```
[the remark this answers is not in the document]
Comment (Ryado) [reply docs/en/interfaces/mysql.md]: ✅

**Pull request merged**